src/nativeAssetManager.js

```diff
@@ -281,40 +282,38 @@ export function newNativeAssetManager(win) {
 
       if (head) win.document.head.innerHTML = replace(head, data);
 
+      let renderPayload = data.assets;
+      if (data.ortb) {
```

Comment:
You are right, `BID_WON` event won't be fired in this case.  
Thanks for noticing this issue.  
My goal was to move firing impression trackers and click trackers from inside PUC.  
I missed this effect: when pbjs is sending impression tracker, at the same time it's marking bid as winning.  

The way I see it, there are two possible solutions:
- mark bid as winning bid (and sending `BID_WON`) when responding to the `allAssetRequest` post message
- or, add one more post message when ad is rendered so that pbjs can mark bid as winning bid.

@dgirardi what do you think?

Reply:
First option is probably cleaner especially if you try to figure out the general case; without safeframe [we set winning bid at the beginning of rendering](https://github.com/prebid/Prebid.js/blob/7cff54d4ebf3c751c86248e9b2c14485aa41025f/src/prebid.js#L441); if that makes sense, it also makes sense to do it on the first message we get, [could be done for this case as well](https://github.com/prebid/Prebid.js/blob/7cff54d4ebf3c751c86248e9b2c14485aa41025f/src/secureCreatives.js#L79) (but I did not check how modules are using `BID_WON`).

Reply:
For native, PUC won't fallback to `window.pbjs.renderAd` even without safe frames.  
Then, I'll go with 1st approach, for this I'll have to update pbjs, more specifically [native ORTB PR](https://github.com/prebid/Prebid.js/pull/7847).